spec/factories/references.rb

```diff
@@ -0,0 +1,10 @@
+require 'faker'
```

Comment:
You don't need these `require 'faker'` lines. You can remove from this and the other files